apm-agent-core/src/main/java/co/elastic/apm/agent/bci/classloading/IndyPluginClassLoader.java

```diff
@@ -40,7 +40,7 @@
  */
 public class IndyPluginClassLoader extends ByteArrayClassLoader.ChildFirst {
     public IndyPluginClassLoader(@Nullable ClassLoader targetClassLoader, ClassLoader agentClassLoader, Map<String, byte[]> typeDefinitions) {
-        super(new MultipleParentClassLoader(Arrays.asList(agentClassLoader, targetClassLoader)), true, typeDefinitions, PersistenceHandler.MANIFEST);
+        super(new MultipleParentClassLoader(agentClassLoader, Arrays.asList(agentClassLoader, targetClassLoader)), true, typeDefinitions, PersistenceHandler.MANIFEST);
```

Comment:
The javadoc for this constructor says about the single-parent argument: "This explicit parent should only be set if the current platform does not allow creating a class loader that extends the bootstrap loader". Do you know why?

Reply:
Not sure. Haven't noticed any issues with providing an explicit parent so far.